src/observer.ts

```diff
@@ -50,21 +50,15 @@ function useObserverReaction(baseComponentName: string) {
     // forceUpdate 2.0
     const forceUpdate = useForceUpdate()
 
-    const lastReaction = useRef<Reaction | undefined>(undefined)
+    const reaction = useRef(
```

Comment:
Is `useRef` really the same as `useMemo`? I suppose it's true in this case since it doesn't depend on anything. Good call.